runtime/browser/demo/index.demo.html

```diff
@@ -22,13 +23,39 @@
       max-height: 736px;
       background-color: white;
     }
+    strategy-explorer {
+      background-color: white;
+      display: none;
+      width: 100%;
+    }
   </style>
 
 </head>
-<body>
+<script>
+function onLoad() {
+  let demoFlow = document.querySelector('demo-flow');
+  let strategyExplorer = document.querySelector('strategy-explorer');
+  demoFlow.strategyExplorer = strategyExplorer;
+}
+function onKeyPressed(event) {
+  if (event.ctrlKey && (event.key == "e")) {
+    let strategyExplorer = document.querySelector('strategy-explorer');
+    let strategyExplorerVisible = strategyExplorer.style.display == 'block';
+    setVisible(strategyExplorer, !strategyExplorerVisible);
 
+    let demoFlow = document.querySelector('demo-flow');
+    setVisible(demoFlow, strategyExplorerVisible);
+  }
+}
+function setVisible(element, isVisible) {
+  element.style.display = isVisible ? 'block' : 'none';
+}
+</script>
+<body onload="onLoad()" onkeydown="onKeyPressed(event)">
```

Comment:
discussed offline and agreed to keep as is for now, and possibly update in the future